keras_nlp/models/xlm_roberta/xlm_roberta_preprocessor.py

```diff
@@ -144,6 +142,14 @@ class XLMRobertaPreprocessor(keras.layers.Layer):
         lambda s1, s2: preprocessor(x=(s1, s2)),
         num_parallel_calls=tf.data.AUTOTUNE,
     )
+
+    # Alternatively, you can create a preprocessor from your own vocabulary.
+    # The usage is exactly the same as above.
+    tokenizer = keras_nlp.models.XLMRobertaTokenizer(proto="model.spm")
```

Comment:
Same comment as for DeBERTa. We need to find a way for this to run or modify somehow